daemon/src/wpaperd.rs

```diff
@@ -50,6 +51,32 @@ impl Wpaperd {
             use_scaled_window,
         })
     }
+
+    pub fn reload_config(&mut self) -> Result<()> {
+        let mut wallpaper_config = self.wallpaper_config.lock().unwrap();
+        let new_config =
+            WallpaperConfig::new_from_path(&wallpaper_config.path).with_context(|| {
+                format!(
+                    "reading configuration from file {:?}",
+                    wallpaper_config.path
+                )
+            });
+        match new_config {
+            Ok(config) => {
+                if config != *wallpaper_config {
+                    *wallpaper_config = config;
+                    println!("Configuration updated");
```

Comment:
I think it's more consistent to use the logging library here, by calling `info!`.